index.html

```diff
@@ -0,0 +1,136 @@
+<!doctype html>
+<html>
+  <head>
+    <title> Aloha Apparel Co</title>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script src="https://use.fontawesome.com/90f704d235.js"></script>
+    <script src="lib/flickity/flickity.min.css"></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="logo">
+        <img src="images/aloha-logo.svg"> 
+      </div>
+          
+      <div class="nav">
+        <a href="google.com">ABOUT</a>
+        <a href="google.com">SHOP</a>
+        <a href="google.com">FEATURED</a>
+        <a href="google.com">UPDATES</a>
+      </div>
+            
+      <div class="shoppingcart">
+        <a href="google.com"> <img src="images/cart-icon.svg"></a>
+      </div>
+
+    </header>
+
+<div class="hero-banner">
+    <p>NEW COLLECTION</p>
+    <h1>Spring/Summer 2016</h1>
+  </div>
+
+<div class="container">
+    <h2>
+      <em>Hand-Crafted</em> <strong>Styles</strong>
+    </h2>
+
+    <p>Aloha Apparel CO. Sells only the finest ready-to-travel clothing. Browse our 
+    current selection of products or sign-up for updates below.</p>
+</div>
+
+<div class="container">
+    <ul>
+      <li>
+        <a href="google.com">MEN'S</a>
+      </li>
+      <li>
+        <a href="google.com">TRENDS</a>
+      </li>
+      <li>
+        <a href="google.com">GLASSES</a>
+      </li>
+      <li>
+        <a href="google.com">SALE</a>
+      </li>
+      <li>
+        <a href="google.com">BAGS</a>
+      </li>
+      <li>
+        <a href="google.com">WOMEN'S</a>
+      </li>
+    </ul>
+</div>
+    
+<div class="container">
+    <h2>
+      <em>Most-Loved</em> <strong>Products</strong>
+    </h2>
+      <ul>
+        <li>
+          <img src="images/checked-stretch-dress.jpg">
+          <p>DRESSES</p>
+          <h2>Checked Stretch Dress</h2>
+         <p>$129.00</p>
+       </li>
+        <li>
+          <img src="images/blue-hipster-backpack.jpg">
+         <p>BAGS</p>
+          <h2>Blue Hipster Backpack</h2>
+         <p>$69.00</p>
+       </li>
+       <li>
+          <img src="images/demin-overall-shorts.jpg">
+          <p>DEMIN</p>
+          <h2>Demin Overall Shorts</h2>
+          <p>$89.00</p>
+        </li>
+        <li>
+          <img src="images/pink-dotted-dress.jpg">
+          <p>DRESSES</p>
+          <h2>Pink Dotted Dress</h2>
+          <p>$99.00</p>
+        </li>
+     
+    </ul>
+</div>
+
+<div class="container">  
+    <h2>
+      <em>Sign-up for </em><strong>Updates</strong>
+    </h2>
+
+    <p>Stay updated on our latest product releases, be first to find out upcoming sales, and get free style tips from our blog. Unsubscribe at any time.</p>
+
+    <form> 
+      Email: <input type="text" email="email"><button>subscribe</button>
+    </form>
+  </div>
+
+
+<footer>
+  <section id="footer left">
+    <small>&copy; 2016 Aloha Apparel Co.<br> Powered by HTML and CSS.
```

Comment:
Make the `<small>`a `<p>`. Don't use break tags, each should be wrapped in a paragraph.